hue/VERSION

```diff
@@ -1 +1 @@
-4.3.0
+4.3.0-gitbase
```

Comment:
Are we planning to support non-gitbase releases? If not, I would just call this `4.3.1` or move away from semantic and call it `4.3.0.1` to be aligned with Hue releases

Reply:
let's call it `4.3.0.1`